developers/requesting-data-commitment-ranges.md

```diff
@@ -0,0 +1,26 @@
+# Requesting data commitment ranges
+
+To request proofs to be submitted to the Blobstream X contract at a
+different cadence, you can do one of the following:
+
+1. [RECOMMENDED] Run the Blobstream X operator with hosted proving, by
```

Comment:
maybe change this description to "Run the Blobstream X operator with hosted proving on the Succinct platform, by running an operator script that pings the platform with proof requests at a specified cadence. Follow the instructions here to run the operator script: ..."

Reply:
https://github.com/celestiaorg/docs/pull/1379/commits/4edf01d4abb0035665a6a1f6c9d89f717381e179